File: backend/src/main/java/com/wecp/progressive/entity/Product.java
package com.wecp.progressive.entity;

public class Product {
private int productid;
private int warehouseid;
private String productName;
private String productDescription;
private int quantity;
private Long price;
public Product() {
}
public Product(int productid, int warehouseid, String productName, String productDescription, int quantity,
        Long price) {
    this.productid = productid;
    this.warehouseid = warehouseid;
    this.productName = productName;
    this.productDescription = productDescription;
    this.quantity = quantity;
    this.price = price;
}
public int getProductid() {
    return productid;
}
public void setProductid(int productid) {
    this.productid = productid;
}
public int getWarehouseid() {
    return warehouseid;
}
public void setWarehouseid(int warehouseid) {
    this.warehouseid = warehouseid;
}
public String getProductName() {
    return productName;
}
public void setProductName(String productName) {
    this.productName = productName;
}
public String getProductDescription() {
    return productDescription;
}
public void setProductDescription(String productDescription) {
    this.productDescription = productDescription;
}
public int getQuantity() {
    return quantity;
}
public void setQuantity(int quantity) {
    this.quantity = quantity;
}
public Long getPrice() {
    return price;
}
public void setPrice(Long price) {
    this.price = price;
}



}

File: frontend/src/app/supplylink/types/Product.ts
<<<<<<< HEAD
import { Warehouse } from "./Warehouse";

export class Product {
    productId: number;
    warehouse: Warehouse;
    productName: string;
    productDescription: string;
    quantity: number;
    price: number;

    constructor(
        productId: number,
        warehouse: Warehouse,
        productName: string,
        productDescription: string,
        quantity: number,
        price: number
    ) {
        this.productId = productId;
        this.warehouse = warehouse;
        this.productName = productName;
        this.productDescription = productDescription;
        this.quantity = quantity;
        this.price = price;
    }

}

=======

import { Warehouse } from './Warehouse';

export class Product {
  
}
>>>>>>> 6eac17dfc6959ce859888e6275c17b3e3a5462e8
